cime_config/config_component.xml

```diff
@@ -404,13 +404,16 @@
 
   <entry id="DRV_RESTART_POINTER">
     <type>char</type>
-    <default_value>rpointer.cpl</default_value>
+    <default_value>rpointer.cpl.${RUN_STARTDATE}-${RUN_REFTOD}</default_value>
     <group>run_begin_stop_restart</group>
     <file>env_run.xml</file>
     <desc>
       Name of the restart pointer file, this can be used to restart from an
       intermediate restart by appending the restart date and time in format YYYY-MM-DD-SSSSS
     </desc>
+    <values>
+       <value>rpointer.cpl.${RUN_STARTDATE}-${RUN_REFTOD}</value>
+    </values>
```

Comment:
This setting of the value seems redundant with the default_value above. Do you actually need both?